src/Uno.UI.FluentTheme/Resources/PriorityDefault/NavigationView_rs1_themeresources_v1.xaml

```diff
@@ -68,6 +69,8 @@
             <StaticResource x:Key="TopNavigationViewItemBackgroundPointerOver" ResourceKey="SystemControlTransparentBrush" />
             <StaticResource x:Key="TopNavigationViewItemBackgroundPressed" ResourceKey="SystemControlTransparentBrush" />
             <StaticResource x:Key="TopNavigationViewItemBackgroundSelected" ResourceKey="SystemControlTransparentBrush" />
+            <StaticResource x:Key="TopNavigationViewItemBackgroundSelectedPointerOver" ResourceKey="SubtleFillColorTransparentBrush" />
```

Comment:
`SubtleFillColorTransparentBrush` doesn't seem to be actually defined anywhere currently in Uno, do we need to add it?